hasura/metadata/databases/default/tables/public_vaults.yaml

```diff
@@ -23,18 +23,6 @@ array_relationships:
         table:
           name: vault_transactions
           schema: public
-remote_relationships:
-  - definition:
-      hasura_fields:
-        - token_address
-        - chain_id
-      remote_field:
-        price_per_share:
-          arguments:
-            chain_id: $chain_id
-            token_address: $token_address
-      remote_schema: vaults
-    name: price_per_share
```

Comment:
this also probably shouldn't be deleted. perhaps this needs to be rebased?

Reply:
We can talk about this later today, I just ran `yarn generate` and this was the result

Reply:
You might have just needed to re-run `yarn hasura metadata reload` again with the dev server running before running generate

Reply:
> You might have just needed to re-run `yarn hasura metadata reload` again with the dev server running before running generate

Tried that now with the same result, topo mentioned yesterday that he'll have a look at the generate script 🙇

Reply:
Ended up fixing these manually with https://github.com/coordinape/coordinape/pull/1687/commits/9231d3344b1ebd0eda28693a8f9ab06b9673b416 and https://github.com/coordinape/coordinape/pull/1687/commits/290da405d9db9eec684d05f7c59634c51d5de769 as the generate script is still not working for me